scripts/utils/start-release.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2025 The Jaeger Authors.
+# SPDX-License-Identifier: Apache-2.0
+#Requires bash version to be >=4. Will add alternative for lower versions
+set -euo pipefail
+
+dry_run=false
+
+PARSED_OPTIONS=$(getopt -n "$0" -o d --long dry-run -- "$@")
```

Comment:
please use the pattern we use elsewere

https://github.com/jaegertracing/jaeger/blob/ce0f5433910471ab7991a4b69b2ee2f152cb68f6/scripts/e2e/spm.sh#L18